### DISCUSSION TITLE
Program shuts down on other OS when PyVista window is opened

### DISCUSSION THREAD

**Discussion by TellusMan** [pyvista/pyvista#3594] [Q&A]
Hi, I have a program that includes PyVista. I have generated .exe file with auto-py-to-exe and installation file with Inno Setup. The program installs and works fine with Windows 11, but when trying to use it with Windows 10 the whole program shuts down the moment when trying to open a PyVista window from a button in the program. So the PyVista part of the program causes the shut down.

Any ideas how to solve the problem?

**Comment by banesullivan** [comment]
I am not experienced with bundling PyVista into exe's presumably with pyinstaller like this. Pinging @akaszynski for input

Though I suspect something is going wrong when linking the OpenGL dependencies and thus causing rendering to segfault.

@TellusMan, can you capture any logs or errors from running the program? Usually VTK dumps an error that will lead you in the right direction

**Reply by TellusMan** [reply]
Sorry I forgot to mention that I tried to open the program from the cmd to see possible error messages, but it gives no messages, just shuts down. Thanks for your input! I'll  investigate the OpenGL situation.
